### PR TITLE
feat(Show Group Work): Display work in preview

### LINKED ISSUE #1032: feat(Show Group Work): Display work in preview
When previewing the VLE, show the student work in Show Group Work components if the previewer generated work for the associated component.

### PR DESCRIPTION
**Pull request by geoffreykwan**
## Changes

Show Group Work now shows the student work created while previewing. It used to show nothing and also throw an error in the console.

## Test

- VLE preview should show the group work created while previewing
- Authoring preview should show "Waiting for response..." message
- Classroom Monitor Step Info preview should show "Waiting for response..." message

Closes #1032

### CODE REVIEW

File: src/assets/wise5/services/studentPeerGroupService.ts [hunk 1]
```diff
@@ -146,4 +146,30 @@ export class StudentPeerGroupService extends PeerGroupService {
...
+    if (this.configService.isPreview()) {
+      const studentWork = [];
+      const latestComponentState = this.studentDataService.getLatestComponentStateByNodeIdAndComponentId(
+        showWorkNodeId,
+        showWorkComponentId
+      );
+      if (latestComponentState != null) {
+        studentWork.push(latestComponentState);
+      }
+      return of(studentWork);
```
Comment (hirokiterashima) [inline src/assets/wise5/services/studentPeerGroupService.ts]: Use ternary expression to simplify code?
```suggestion
      const latestComponentState = this.studentDataService.getLatestComponentStateByNodeIdAndComponentId(
        showWorkNodeId,
        showWorkComponentId
      );
      return latestComponentState != null ? of([latestComponentState]) : of([]);
```

**Review by hirokiterashima — COMMENTED** [review]
Functionality works.

Should the ```retrieveStudentWork()``` function be in StudentPeerGroupService or PeerGroupService? It's being called by authors and graders right now.

Also, see inline code comments.

**Comment by geoffreykwan** [comment]
`retrieveStudentWork()` is only being called by show-group-work-student.component.ts right now so it's only being called in the student context. It's a little confusing because the teacher can launch dialogs that display the student views.

**Review by hirokiterashima — APPROVED** [review]
LGTM

**Pull request merged**